sql/src/test/java/io/crate/metadata/RoutingTest.java

```diff
@@ -64,4 +64,44 @@ public void testStreamingWithoutLocations() throws Exception {
         Routing routing2 = Routing.fromStream(in);
         assertThat(routing1.locations(), is(routing2.locations()));
     }
+
+    @Test
+    public void testUpdateLocations() {
```

Comment:
maybe make the method name more specific, what aspect of testUpdateLocations is this testing?